style.css

```diff
@@ -0,0 +1,151 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+* *::before *::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  margin: 10px;
+  padding: 10px;
+  border: 20px;
+  background-image: url(Header.svg);
+  max-height: 100vh;
+  max-width: auto;
+}
+
+a {
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 10px;
+  letter-spacing: 2px;
+  text-decoration: none;
+  background: none;
+  padding: 5px 5px;
+  display: inline-block;
+  width: 100%;
+  text-align: center;
+}
+
+a:hover {
+  background-color: #6070ff;
+}
+
+a:enabled {
+  background-color: #fff;
+  color: #7f8cff;
+}
+
+a:disabled {
+  background-color: #5e6c84;
+}
+
+a:active {
+  background: #2230d2;
+}
+
+.toggleNav {
+  display: none;
+}
+
+img {
+  width: 10%;
+  height: auto;
+}
+
+h1 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 48px;
+  font-weight: 800;
+  line-height: 60px;
+}
+
+h6 {
+  color: #7f8cff;
+  font-size: 16px;
+}
+
+.logo {
+  /* border: 1px solid orange; */
+  display: flex;
+  margin-left: 100px;
+}
+
+li {
+  list-style-type: none;
+}
+
+.header {
+  /* border: 1px solid yellow; */
+  display: flex;
+  margin: 0;
+  padding: 0;
+  justify-content: space-between;
+}
+
+.flex-nav ul {
+  /* border: 1px solid black; */
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  /* border: 1px solid green; */
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  justify-content: center;
+}
+
+.introduction {
+  /* border: 1px solid red; */
+  width: 60%;
+  margin-left: calc(60% - 500px);
+}
+
+.info ul {
+  /* border: 1px solid blue; */
+  display: flex;
+  color: #505f79;
+  width: 60%;
+  margin-left: calc(60% - 500px);
+}
+
+.media {
+  width: 30px;
+}
```

Comment:
- [ ] The social icons are too small for a clickable element, kindly consider fixing the size of the social account icons so they align with the design provided in the Figma design.

## Your Design
![image](https://user-images.githubusercontent.com/68030297/219701044-878cab22-c9a1-45c4-870d-f9ec1b97ddd4.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/219701129-19758f0b-b482-47ce-9041-0db15dce0a2d.png)

Reply:
- [ ] Kindly implement the change requested by the previous reviewer.